simbad/command_line/__init__.py

```diff
@@ -261,7 +261,7 @@ def _argparse_mr_options(p):
                     help='Path to the MR program to use. Options: < molrep | phaser >')
     sg.add_argument('-refine_program', type=str, default="refmac5",
                     help='Path to the refinement program to use. Options: < refmac5 >')
-    sg.add_argument('-refine_cycles', type=int, default=30,
+    sg.add_argument('-refine_cycles', type=int, default=None,
```

Comment:
No need to define `default=None`, this is automatically defined. Maybe add to the docstring `[default: 30]`